Parser/pegen/pegen.h

```diff
@@ -14,6 +14,11 @@ enum INPUT_MODE {
 };
 typedef enum INPUT_MODE INPUT_MODE;
 
+enum START_RULE {
```

Comment:
Why this enum of only one field?

Reply:
We'll probably add more, we we need to choose between start rules. I'm currently working on maybe changing the start rule for f-string to `expressions_rule` and for this I added another field `EXPRESSIONS` to this enum.

Reply:
(Probably for a separate PR.)

Maybe we could pass a string literal instead? By convention we could have `NULL` -> `start` and a few other names could be recognized based on metadata given in the .gram file, e.g. [1]
```
@alternate_starts "single_input file_input eval_input"
```
which would generate a little table at the end
```
static struct { char *, mod_ty (*)(Parser *) } [] alternate_starts = {
    {"single_input", single_input_rule},
    ...
    NULL,
};
```
and then the dispatch function could just look up the argument in the table. (This is a negligible cost given everything else that goes into parsing even one line.)

____
[1] We'd need to add a few new rules named `single_input`, `file_input`, `eval_input` that correspond to those targets in Grammar/Grammar.